test/extensions/filters/http/ext_authz/ext_authz_integration_test.cc

```diff
@@ -248,6 +248,10 @@ class ExtAuthzHttpIntegrationTest : public HttpIntegrationTest,
         {":scheme", "http"},
         {":authority", "host"},
         {"x-case-sensitive-header", case_sensitive_header_value_},
+        {"baz", "foo"},
+        {"bat", "foo"},
+        {"x-append-bat", "append-foo"},
+        {"x-append-bat", "append-bar"},
```

Comment:
Oops, sorry. These two entries should not be here. This is the original request.